lib/handlers/DeleteHandler.js

```diff
@@ -11,22 +11,16 @@ class DeleteHandler extends BaseHandler {
      * @param  {object} res http.ServerResponse
      * @return {function}
      */
-    send(req, res) {
+    async send(req, res) {
         const file_id = this.getFileIdFromRequest(req);
-        if (!file_id) {
-            console.warn('[DeleteHandler]: not a valid path');
-            return Promise.resolve(super.send(res, 404, {}, 'Invalid path name\n'));
+        if (file_id === false) {
+            throw ERRORS.INVALID_PATH;
```

Comment:
Other handles return ERRORS.INVALID_PATH if file_id === false. Should this be done here as well to keep it uniform?